com.unity.multiplayer.mlapi/Runtime/NetworkVariable/Collections/NetworkDictionary.cs

```diff
@@ -2,11 +2,8 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.IO;
-using Unity.Multiplayer.Netcode.Timing;
-using Unity.Multiplayer.Netcode.Serialization.Pooled;
-using Unity.Multiplayer.Netcode.Transports;
 
-namespace Unity.Multiplayer.Netcode.NetworkVariable.Collections
+namespace Unity.Multiplayer.Netcode
```

Comment:
we might still want to keep network collections behind a namespace like `Unity.Multiplayer.Netcode.Collections`, what would you think @mattwalsh-unity ?

Reply:
actually, following the same thought I stated in the PR description, we might not want to hide these types behind a namespace. I'd vote to keep this under `Unity.Multiplayer.Netcode` but I'd like you to make the final call.